frontend/src/static/js/main.js

```diff
@@ -188,17 +188,38 @@ window.app = new window.Vue({
     deactivateTab() {
       this.isTabActive = false;
       window.addHash('tabOpen', this.isTabActive);
-      window.removeHash('tabAuthor');
-      window.removeHash('tabRepo');
       window.removeHash('tabType');
+      this.removeZoomHashes();
       window.encodeHash();
     },
 
+    removeAuthorshipHashes() {
+      window.removeHash('tabAuthor');
+      window.removeHash('tabRepo');
+    },
+
+    removeZoomHashes() {
+      window.removeHash('zoomSince');
+      window.removeHash('zoomUntil');
+      window.removeHash('repoIndex');
+      window.removeHash('userIndex');
+      window.removeHash('avgCommitSize');
+      window.removeHash('zoomFilterGroupSelection');
+      window.removeHash('zoomFilterTimeFrame');
```

Comment:
Sorry haven't been following for a while, whats the diff between `zoomFilterTimeFrame` and `zoomSince` + `zoomUntil`?

Reply:
`zoomFilterTimeFrame` refers to the `granularity` of each ramp slice, `zoomSince` and `zoomUntil` refers to the since and until date of the commits in zoom view.

All these settings are recorded at the moment when user opens the zoom tab, to make sure that changes (in v_summary) after the opening the zoom tab won't affect the zoom tab.

Reply:
I see, just realized that the same convention is being used on summary. Maybe we should rename it to match the fields that they're intended for to avoid confusion.